scripts/broker.csx

```diff
@@ -0,0 +1,104 @@
+#r "System.Data"
+#r "../src/packages/RabbitMQ.Client.4.1.1/lib/net451/RabbitMQ.Client.dll"
```

Comment:
Having to reference the file like this is unfortunate, since the next time we update the client we'll have to remember to manually update this.